app/models/energy/ScheduleFixedInterval.py

```diff
@@ -0,0 +1,63 @@
+"""Schedule Fixed Interval Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError, UrlStr, ConstrainedStr
+from typing import List, Union
+from enum import Enum
+from uuid import UUID, uuid4
+import datetime
+from app.models.energy.ScheduleBase import ScheduleContinuous, ScheduleDiscrete, ScheduleNumericType, ScheduleUnitType, Date, Time
+
+
+class ScheduleFixedInterval(BaseModel):
+    """Used to specify a start date and a list of values for a period of analysis."""
+
+    type: Enum('ScheduleFixedInterval', {'type': 'ScheduleFixedInterval'})
+
+    name: str = Schema(
+        ...,
+        regex=r'^[\s.A-Za-z0-9_-]*$',
+    )
+
+    start_date: Date
+
+    is_leap_year: bool = Schema(
+        False,
+        description='A boolean to indicate if datetime is for a leap year. Default is'
+        ' `false`.'
+    )
+
+    values: List[int] = Schema(
+        ...,
+        minItems=24,
+        maxItems=8784,
+        description='A list of hourly values for the simulation.'
+    )
+
+    @validator('values', whole=True)
```

Comment:
This validation and the next one should be compiled into one. You can check if items are in range and that will be it.